### ISSUE TITLE
Translate `setval` to `auto_increment` syntax

### ISSUE THREAD

**Issue by coffeegoddd** [dolthub/pg2mysql#42]
```
SELECT pg_catalog.setval('public.my_table_id_seq', 33, true);
```

Above statement remains in stdout output, but MySQL does not support `setval` function.  This errors with:
```
SELECT pg_catalog.setval('public.my_table_id_seq', 33, true): function: 'setval' not found
function: 'setval' not found
```

Should be translated to `auto_increment` syntax.

**Comment by coffeegoddd** [comment]
mysql syntax `ALTER TABLE my_table AUTO_INCREMENT = value;`

**Comment by timsehn** [comment]
Resolved by #43
